api/desecapi/views.py

```diff
@@ -118,6 +120,19 @@ def perform_destroy(self, instance: models.Domain):
                 parent_domain.update_delegation(instance)
 
 
+class SerialList(generics.ListAPIView):
+    permission_classes = (IsVPNClient,)
+    throttle_classes = []  # don't break slaves when they ask too often (our cached responses are cheap)
+
+    def list(self, request):
```

Comment:
this method contains a lot of boilerplate caching code, is there a better way to achieve this?

Reply:
no